lib/util.js

```diff
@@ -279,11 +279,15 @@ const util = {
     util.run('ninja', ['-C', config.outputDir, config.buildTarget, '-k', num_compile_failure], options)
   },
 
-  lint: (options = {}) => {
+  lint: (options) => {
+    if (!options) {
+      options = 'origin/master';
```

Comment:
why would `options` be the base_branch? I think this should still be a map

Reply:
We are passing the `base_branch` as an option here: https://github.com/brave/brave-browser/pull/3346/files#diff-1bb8ffaf71730e8067757db1a6a153dbR133. We can use a map as well, but since there is only one option I'm not sure if there will be any difference. Let me know.

Reply:
but does it make sense to call something `options` if it's actually the `base_branch`? Either way I think this should continue to be a map for potential future options